Upstream: No
Reason: Support the version of py3-praw present in Alpine Linux aports

diff --git a/requirements.txt b/requirements.txt
index 0dc7ce4..b2db52f 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 xmltodict<0.12.0; python_version == '3.3'
 xmltodict==0.12; python_version != '3.3'
 pytz
-praw>=4.0.0,<6.0.0
+praw>=4.0.0,<7.1.0
 # transitive dependency of praw; v0.18 introduced f-string syntax
 update-checker<0.18; python_version < '3.6'
 geoip2<3.0; python_version <= '3.5' and python_version != '2.7'
